dev/helpers/mssql/run_migrations.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-# There seems to be [a bug with docker-compose](https://github.com/docker/compose/issues/4076#issuecomment-324932294) 
+# There seems to be [a bug with docker-compose](https://github.com/docker/compose/issues/4076#issuecomment-324932294)
 # where it takes ~40ms to connect to the terminal output of the container, so stuff logged to the terminal in this time is lost.
 # The best workaround seems to be adding tiny delay like so:
 sleep 0.1;
 
 MIGRATE_DIRECTORY="/mnt/migrator/DbScripts"
+HELPERS_DIRECTORY="/mnt/helpers"
```

Comment:
💭 Won't need this if you use a temp file.

Reply:
I added the temp file but it's still used in 2 locations, seems it useful to keep.